config/ext/capybara.yml

```diff
@@ -17,7 +17,7 @@ Capybara:
 
 Capybara/ClickLinkOrButtonStyle:
   Description: Checks for methods of button or link clicks.
-  Enabled: pending
+  Enabled: false # adds noise which actually does not improve much
```

Comment:
Please also link the issue, where we discussed this cop in this comment. So the next person coming across the cop knows, there has already been a discussion